### ISSUE TITLE
dicom-microscopy extension needs to pass supported media types

### ISSUE THREAD

**Issue by bcarthic** [OHIF/Viewers#3785]
### Describe the Bug

Currently dicom-microscopy [extension](https://github.com/OHIF/Viewers/blob/8c2f495405599924fb497d93cb9f87a80d8b8dd4/extensions/dicom-microscopy/src/utils/dicomWebClient.ts#L6) passes `*/*` as accept header while retrieving frames.

```
DICOMwebClient._buildMultipartAcceptHeaderFieldValue = () => {
  return '*/*';
};
```

As a DICOM web service, if we see `*/*`, we will return back the original transfer syntax from the DICOM file, in this case, if the original transfer syntax is used, the OHIF microscopic viewer won't display the images. Shouldn't it pass the list of supported types which is being passed to the method?

```
       var supportedMediaTypes = {
          '1.2.840.10008.1.2.1': ['application/octet-stream'],
          '1.2.840.10008.1.2.5': ['image/x-dicom-rle'],
          '1.2.840.10008.1.2.4.50': ['image/jpeg'],
          '1.2.840.10008.1.2.4.51': ['image/jpeg'],
          '1.2.840.10008.1.2.4.57': ['image/jpeg'],
          '1.2.840.10008.1.2.4.70': ['image/jpeg'],
          '1.2.840.10008.1.2.4.80': ['image/x-jls', 'image/jls'],
          '1.2.840.10008.1.2.4.81': ['image/x-jls', 'image/jls'],
          '1.2.840.10008.1.2.4.90': ['image/jp2'],
          '1.2.840.10008.1.2.4.91': ['image/jp2'],
          '1.2.840.10008.1.2.4.92': ['image/jpx'],
          '1.2.840.10008.1.2.4.93': ['image/jpx']
        };
        var headers = {
          Accept: DICOMwebClient._buildMultipartAcceptHeaderFieldValue(mediaTypes, supportedMediaTypes)
        };
```

### Steps to Reproduce

1. Use the microscopic viewer to display digital pathology images
2. Verify if the images are loaded if the accept header is `*/*`

### The current behavior

Current behavior, with accept header `*/*`, the original transfer syntax is returned back.

### The expected behavior

As a DICOM web service we expect a list of supported types similar to how SLIM viewer does. So it should pass the list of supported types which it has already.

### OS

Windows 11

### Node version

18.16.1

### Browser

Edge Version 119.0.2151.58 (Official build) (64-bit)

**Comment by wayfarer3130** [comment]
What transfer syntax is it returning that the viewer can't decode/display?  Usually one wants the original transfer syntax to be returned, or at least image/* because transcoding the images server side often results in unacceptable performance of the viewer.

**Comment by sedghi** [comment]
I think this is related issue for WSI not rendering

**Comment by wayfarer3130** [comment]
> I think this is related issue for WSI not rendering

No, this issue has nothing to do with WSI not rendering.  That problem is that the response content type is wrong for the bulkdata, and the dicomweb client has no way to force the response type to a specific value to over-ride the server behaviour.

This issue is about the request being made with a too general accept type, and ending up getting a response type the client can't handle.  The over-ride is in DicomWebClient that sets this and needs to be configurable.  The problem is, that one also over-rides requests for bulkdata, so it needs to be more specific to request an image type.
